--- orig.sh	2017-09-14 22:42:05.210893044 +0200
+++ modified.sh	2017-09-14 22:47:59.877681421 +0200
@@ -65,6 +65,7 @@
 	default_vhost
 	hipe_compile
 	vm_memory_high_watermark
+	cluster_partition_handling
 )
 fileConfigKeys=(
 	management_ssl_cacertfile
@@ -213,7 +214,7 @@
 
 		local rawVal=
 		case "$conf" in
-			verify|fail_if_no_peer_cert|depth)
+			verify|fail_if_no_peer_cert|depth|cluster_partition_handling)
 				[ "$val" ] || continue
 				rawVal="$val"
 				;;
